source/change-streams/change-streams.rst

```diff
@@ -155,6 +155,46 @@ If the aggregation with ``$changeStream`` specified completes successfully, the
     removedFields: Array<String>;
   }
 
+The responses to a change stream aggregate or getMore have the following structures:
+
+.. code:: typescript
+
+  /**
+   * Response to a successful aggregate.
+   */
+  {
+      ok: 1,
+      cursor: {
+         ns: String,
+         id: Int64,
+         firstBatch: Array<ChangeStreamDocument>
+      },
+      operationTime: Timestamp,
+      $clusterTime: Document,
+      /**
+       * postBatchResumeToken is returned in MongoDB 4.2 and later.
+       */
+      postBatchResumeToken: Document
+  }
+
+  /**
+   * Response to a successful getMore.
+   */
+  {
+      ok: 1,
+      cursor: {
+         ns: String,
+         id: Int64,
+         nextBatch: Array<ChangeStreamDocument>
+      },
+      operationTime: Timestamp,
+      $clusterTime: Document,
+      /**
+       * postBatchResumeToken is returned in MongoDB 4.2 and later.
```

Comment:
Fine with me to merge this, but we may have to ammend this to MongoDB 4.2 and later, MongoDB 4.0.x and later, etc. depending on how backports go.